pyproject.toml

```diff
@@ -105,6 +101,94 @@ test = [
     "coverage",
 ]
 
+[tool.ruff]
+target-version = "py38"  # minimum supported version
+line-length = 88  # same as Black.
+select = [
+    "A",  # flake8-builtins
+    "ANN",  # flake8-annotations
+    "ARG",  # flake8-unused-arguments
+    "B",  # flake8-bugbear
+    "BLE",  # flake8-blind-except
+    "C4",  # flake8-comprehensions
+    "D",  # pydocstyle
+    "E",  # pycodestyle error
+    "EM",  # flake8-errmsg
+    "ERA",  # eradicate
+    "F",  # Pyflakes
+    "FA",  # flake8-future-annotations
+    "FBT",  # flake8-boolean-trap
+    "FIX",  # flake8-fixme
+    "FLY",  # flynt
+    "FURB",  # refurb
```

Comment:
I get the following warning with this ruleset.
`warning: Selection `FURB` has no effect because the `--preview` flag was not included.`  Are you getting that too?

Reply:
`--preview` is for the rules that are still being workshopped. This one used to be fine, but got moved there—i had used it on my other project and it was real nice.

It might be annoying, though, when this is removed and you have to fix the issues in the middle of a different PR. I'll comment it out like you had, in a "we want this someday but not yet" section.